frontend/src/modules/entitydetails/components/EntityDetails.test.js

```diff
@@ -96,12 +96,12 @@ function createEntityDetailsWithStore() {
 
 
 describe('<EntityDetailsBase>', () => {
-    beforeAll(() => {
+    beforeEach(() => {
         sinon.stub(editor.actions, 'updateFailedChecks').returns({ type: 'whatever'});
         sinon.stub(editor.actions, 'resetFailedChecks').returns({ type: 'whatever'});
     });
 
-    afterAll(() => {
+    afterEach(() => {
```

Comment:
I think that's the wrong approach. Those were fine. I believe what you want to do is add a third step, `afterEach`, that calls `.reset()`. Otherwise, you mock and de-mock the object after every test. I have no idea if that has an impact in terms of performance, but I'm pretty sure that's not the philosophy of `sinon`.

Reply:
I tried that first, but it gave errors:
* `TypeError: editor.actions.updateFailedChecks.restore is not a function`
* `Received: undefined`

Reply:
I've just tested locally with this code and it passes: 

```js
    beforeAll(() => {
        sinon.stub(editor.actions, 'updateFailedChecks').returns({ type: 'whatever'});
        sinon.stub(editor.actions, 'resetFailedChecks').returns({ type: 'whatever'});
    });

    afterEach(() => {
        editor.actions.updateFailedChecks.reset();
        editor.actions.resetFailedChecks.reset();
    });

    afterAll(() => {
        editor.actions.updateFailedChecks.restore();
        editor.actions.resetFailedChecks.restore();
    });
```